src/pages/careers.tsx

```diff
@@ -0,0 +1,287 @@
+                        <p>
+                            We started developing Gitpod about 2 years ago and this April we officially launched it. Gitpod is a very young product which gives you lots of opportunities to shape its future. Currently we’re working in a team of less than 20 people and we’re planning to grow continually.
+                        </p>
+
+
+                        <p>
+                            Now we are looking for a passionate sales representative who takes control of our inbound marketing. You should have an understanding of Gitpod’s technical background and like to engage with industrial customers. We especially support remote work and collaboration from different time zones.
+                        </p>
+
+                        <h4>What you’ll do</h4>
+
+                        <ul>
+                            <li>engage with industrial customers to find the best solutions and prices</li>
+                            <li>follow-up on leads from inbound marketing and conferences</li>
+                            <li>tailor demonstrations for customers, conferences, and special events</li>
+                            <li>work closely with the customer to set up and maintain a successful demo period, be available to answer questions and trouble-shoot as needed</li>
+                            <li>confer with customers and engineers to assess equipment needs and to determine system requirements</li>
+                        </ul>
+
+                        <h4>
+                            What we’re looking for
+                        </h4>
+
+                        <ul>
+                            <li>ability to creatively explain and present complex concepts in an easy to understand manner</li>
+                            <li>solid technical background with hands-on experience in software development and web technologies</li>
+                        </ul>
+
+                        <h4>
+                            Contact
+                        </h4>
+
+                        <p>
+                            Would you like to join our team? Then we’re looking forward to hearing from you. Please send your application, including your motivation and your CV to career@typefox.io
+                        </p>
+
+                    </Job>
+
+                    <Job>
+                        <h3>Software Engineer Developer Tools</h3>
+                    </Job>
+
+                    <Job>
+                        <h3>Operation and Sales Manager</h3>
+                    </Job>
+
+                    <Job>
+                        <h3>Solution Engineer</h3>
+                    </Job>
+
+                    <p style={{margin: "70px 0 40px 0"}}>
+                        If no current Job offer fits your qualifications, but you're totally in love with Gitpod and want to help us improving it,
+                        please send your application at careers@typefox.io anyway. We're curious to hear about you.
```

Comment:
Please transform the email into a `mailto` link:

```suggestion
                        please send your application at <a href="mailto:careers@typefox.io">careers@typefox.io</a> anyway. We're curious to hear about you.
```